Changelog.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to
 
 ## Unreleased
 
+### Changed
+
+-   `scripts/nordic-publish.js` is now transpiled and bundled on install, so we
```

Comment:
Did you intend to delete the file from the git repo?

Reply:
Yes, that was my intention but I failed to do so. :-) Seems I messed it up because I also added it to `.gitignore`. Thanks for noticing, fixed it in 39f87c8179.